syntax = "proto3";

package rdk.rpc.wifi;

option cc_generic_services = true;

message CancelWPSPairingResponse {
  bool success = 1;
  string error = 2;
}

message CancelWPSPairingRequest {
  string interface_name = 1;
}

service WiFiService {
  rpc CancelWPSPairing(CancelWPSPairingRequest) returns (CancelWPSPairingResponse);
}
